app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -215,6 +211,16 @@ def finish(*_args)
 
   alias_method :abort_job, :cleanup
 
+  def cancel(*args)
+    _log.info "Job Canceling"
+    if self.state != "canceling" # ensure change of states
+      signal :cancel
+    else
+      unqueue_all_signals
+      queue_signal(:abort_job, "Job Canceled", "cancel", *args)
```

Comment:
I updated to use ```process_cancel``` and also ```process_finished```. I also added a screenshot to show how it looks like when a job is canceled now.